src/components/AddItem.js

```diff
@@ -1,11 +1,102 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { db } from '../lib/firebase';
+
+const initialFormState = {
+  itemName: '',
+  timeFrame: 7,
+  lastPurchased: null,
+};
 
 const AddItem = () => {
+  const token = 'user token';
```

Comment:
This is just a placeholder for the user token for now. @veeteeran @LINDGRENBA if you can pass the userToken as props to AddItem then we can grab it and use it.